### ISSUE TITLE
Postman Console Crashes on expanding a log to show global variable value

### ISSUE THREAD

**Issue by hari2anand** [a85/POSTMan-Chrome-Extension#6306]
Tried to print a couple of the global variables using console log, which gets printed as below screenshot but expanding the same crashes the Postman console

Code snippet used for logging,
console.log("Validation for Items: ",pm.globals.get("item1")," ; ", pm.globals.get("item2"));

![image](https://user-images.githubusercontent.com/45207514/56198635-8dcefb80-603b-11e9-8970-1ea3bcd5b5fe.png)
![image](https://user-images.githubusercontent.com/45207514/56198663-9aebea80-603b-11e9-8699-4e3c0ea84316.png)

**Comment by numaanashraf** [comment]
@hari2anand Please follow the issue template and guidelines.

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 6.0.1]
 - OS: [e.g. macOS High Sierra 10.13.2]

**Additional context**
Add any other context about the problem here.

**Comment by ronakodhaviya** [comment]
Describe the bug
When you print an environment variable appending a string in front of it, console breaks. 
 
To Reproduce
Steps to reproduce the behavior:
1. In any request's test script add following line,
`console.log('foo', pm.environment.get("foo"));`
2. run the request and check the console for value of environment variable "foo"
3. Console breaks 

Expected behavior
Console should not break. 

Screenshots
<img width="739" alt="Screenshot 2019-04-17 at 11 58 03 AM" src="https://user-images.githubusercontent.com/25086676/56265793-259a1b80-6108-11e9-8f35-1a3c55615c84.png">

App information (please complete the following information):
Postman v7.0.8 on MacOS Mojave 10.14.3

App Type [Native App]

Additional context
Having this problem since past one month.

**Comment by hari2anand** [comment]
> Describe the bug
> When you print an environment variable appending a string in front of it, console breaks.
> 
> To Reproduce
> Steps to reproduce the behavior:
> 
> 1. In any request's test script add following line,
>    `console.log('foo', pm.environment.get("foo"));`
> 2. run the request and check the console for value of environment variable "foo"
> 3. Console breaks
> 
> Expected behavior
> Console should not break.
> 
> Screenshots
> <img alt="Screenshot 2019-04-17 at 11 58 03 AM" width="739" src="https://user-images.githubusercontent.com/25086676/56265793-259a1b80-6108-11e9-8f35-1a3c55615c84.png">
> 
> App information (please complete the following information):
> Postman v7.0.8 on MacOS Mojave 10.14.3
> 
> App Type [Native App]
> 
> Additional context
> Having this problem since past one month.
@misteronak 👍 
Thanks for the help, as was away from the system for a while. Thanks for your time in updating the above. :-)

**Comment by p00j4** [comment]
```Postman version: 7.0.9```
I encountered this 

1. Without console statement too.
**Steps to reproduce:** Make a sendRequest call and try to expand it (request of any in console.
```
// url = "GIVE URL"
// data = "GIVE JSON DATA"
pm.sendRequest({
        url: login_url,
        method: 'POST',
        body: {
            mode: 'raw',
            raw: data
        }
    }, function (err, res) {
        if (err){ throw err;}
        let response =  res.json();
    });
```

2. With calling a global function (eval(globas.my_fun()) which has console statement. 

This has become a huge bottleneck in debugging

**Comment by codenirvana** [comment]
> With calling a global function (eval(globas.my_fun()) which has console statement.

@p00j4 `console` in eval might not work, Refer https://github.com/postmanlabs/postman-app-support/issues/5942#issuecomment-466961626

**Comment by p00j4** [comment]
@codenirvana thanks, however my console logs are working perfectly i.e. prints are working. It's the request which on expand click, crashes the console window app.
Also there are 2 different cases (without console too)

**Comment by appurva21** [comment]
@p00j4 
Can you send us the screenshots from:
**View -> Developer -> Show DevTools (Current View) -> Console Tab**
for the Postman console window after it crashes.

**Comment by harryi3t** [comment]
@appurva21 Was able to reproduce this by:
 1. Send a request. Postman Console should already be open.
 2. Expand the request in the console and select `raw` mode
![image](https://user-images.githubusercontent.com/5207331/61705663-e234f600-ad63-11e9-8eb1-555777acc692.png)
3. Send another request with a script `console.log(1,2,3)`
3. Expand the log, the console would crash

Also, that would mean this would not happen if the `mode` selected for the request is `pretty`.
- cc @hari2anand

**Comment by codenirvana** [comment]
@hari2anand We've pushed a fix for this on `v7.7.2`. Can you update your app and confirm if that fixes your issue?
